### PR TITLE
Add documentation for credentials API

### PR DESCRIPTION
**Pull request by garethbowen**
medic/cht-core#5904

### CODE REVIEW

File: content/en/apps/guides/messaging/gateways/africas-talking.md [hunk 1]
```diff
@@ -32,27 +32,7 @@ Then generate an "API Key" (we'll refer to this as the `at-api-key`) and save th
...
-
-
-
-You can also add it via Fauxton:
- - Navigate to [the Config screen](http://localhost:5984/_utils/#/_config)
- - Click `Add Option`
- - The `Section` should be `medic-credentials`, the `Name` should be `africastalking.com:incoming` or `africastalking.com:outgoing`, and the value should be the relevant API key.
- - Click `Create`
- - You should then be able to see your credential in the list of configuration shown
+The Africa's Talking integration uses the CHT Credentials service to retrieve the API keys using the IDs `africastalking.com:incoming` and `africastalking.com:outgoing`. Use the [CHT credentials API](/apps/reference/api#put-apiv1credentials) to securely store the credentials.
```
Comment (m5r) [inline content/en/apps/guides/messaging/gateways/africas-talking.md:35]: Has the migration been done yet? If not, where is it being tracked?

**Review by m5r — APPROVED** [review]
Nice work! LGTM

File: content/en/apps/guides/messaging/gateways/africas-talking.md (hunk 1 above)
Comment (garethbowen) [reply content/en/apps/guides/messaging/gateways/africas-talking.md:35]: Unfortunately I couldn't figure out a way to do migration. As part of the 4.0.0 uplift we'll have to create a new clustered database which means the credentials from the old database will no longer be available. I've made a note that this is a manual step in the 4.0.0 upgrade process.

Comment (m5r) [reply content/en/apps/guides/messaging/gateways/africas-talking.md:35]: Makes sense, thanks for the explanation :+1:

**Pull request merged**